classes/cyclonedx-export.bbclass

```diff
@@ -161,9 +167,7 @@ def append_to_vex(d, cve, cves, bom_ref):
     Collect CVE status information from within open embedded recipes and append to add to cve dictionary.
     This could be backported CVE fixes or ignored CVEs.
     """
-    from oe.cve_check import decode_cve_status
-
-    decoded_status, state, justification = decode_cve_status(d, cve)
+    cve, decoded_status, state, justification = cve
```

Comment:
The variable naming here is not ideal and might cause confusion.

We assign the variables `cve`, ... to the value of an already assigned variable `cve`.